pkgs/dart_services/lib/src/generative_ai.dart

```diff
@@ -83,90 +104,61 @@ ${column != null ? 'COLUMN: $column\n' : ''}
 SOURCE CODE:
 $source
 ''';
-    final stream = model.generateContentStream([Content.text(prompt)]);
+    final content = Content(
+      parts: [
+        systemInstructions,
```

Comment:
FWIW: The latest version does support system instructions https://pub.dev/documentation/google_cloud_aiplatform_v1beta1/latest/aiplatform/GenerateContentRequest/systemInstruction.html

Reply:
👍 cool, that's worth switching to in this PR